packages/abstractions/src/authentication/authenticationProvider.ts

```diff
@@ -1,4 +1,4 @@
-import { RequestInformation } from "../requestInformation";
+import type { RequestInformation } from "../requestInformation";
```

Comment:
The eslint rule added seems to suggest that when dealing with an interface then all the imports should be declared using the `import type` syntax I can change the settings to warning then revert to the conventional import syntax

Reply:
I don't care much about the rule itself. I want to make sure we get it right, especially when we consider the generated code.

It is suggesting this because we're only using that import for a parameter type and in effects not calling any code from that module?

Reply:
ok understood